examples/using_fhir_views.ipynb

```diff
@@ -0,0 +1,671 @@
+      "Requirement already satisfied: protobuf~=3.19 in /usr/local/google/home/omarismail/.local/lib/python3.10/site-packages (from google-fhir-views[r4,spark]) (3.20.2)\n",
+      "Requirement already satisfied: pandas~=1.1 in /usr/local/google/home/omarismail/.local/lib/python3.10/site-packages (from google-fhir-views[r4,spark]) (1.5.0)\n",
+      "Requirement already satisfied: google-fhir-core~=0.9.2 in /usr/local/google/home/omarismail/.local/lib/python3.10/site-packages (from google-fhir-views[r4,spark]) (0.9.2)\n",
+      "Requirement already satisfied: python-dateutil~=2.8 in /usr/local/google/home/omarismail/.local/lib/python3.10/site-packages (from google-fhir-views[r4,spark]) (2.8.2)\n",
+      "Requirement already satisfied: requests~=2.27 in /usr/lib/python3/dist-packages (from google-fhir-views[r4,spark]) (2.28.1)\n",
+      "Requirement already satisfied: requests-mock~=1.9 in /usr/local/google/home/omarismail/.local/lib/python3.10/site-packages (from google-fhir-views[r4,spark]) (1.10.0)\n",
+      "Requirement already satisfied: immutabledict~=2.2 in /usr/local/google/home/omarismail/.local/lib/python3.10/site-packages (from google-fhir-views[r4,spark]) (2.2.1)\n",
+      "Requirement already satisfied: google-fhir-r4~=0.9.2 in /usr/local/google/home/omarismail/.local/lib/python3.10/site-packages (from google-fhir-views[r4,spark]) (0.9.2)\n",
+      "Requirement already satisfied: stringcase==1.2.0 in /usr/local/google/home/omarismail/.local/lib/python3.10/site-packages (from google-fhir-core~=0.9.2->google-fhir-views[r4,spark]) (1.2.0)\n",
+      "Requirement already satisfied: antlr4-python3-runtime~=4.9.3 in /usr/local/google/home/omarismail/.local/lib/python3.10/site-packages (from google-fhir-core~=0.9.2->google-fhir-views[r4,spark]) (4.9.3)\n",
+      "Requirement already satisfied: numpy>=1.21.0 in /usr/lib/python3/dist-packages (from pandas~=1.1->google-fhir-views[r4,spark]) (1.24.2)\n",
+      "Requirement already satisfied: pytz>=2020.1 in /usr/lib/python3/dist-packages (from pandas~=1.1->google-fhir-views[r4,spark]) (2022.7.1)\n",
+      "Requirement already satisfied: six>=1.5 in /usr/lib/python3/dist-packages (from python-dateutil~=2.8->google-fhir-views[r4,spark]) (1.16.0)\n"
+     ]
+    }
+   ],
+   "source": [
+    "!pip3 install google-fhir-views[r4,spark]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas\n",
+    "\n",
+    "from sqlalchemy import dialects\n",
+    "from sqlalchemy import engine\n",
+    "\n",
+    "from google.fhir.views import r4\n",
+    "from google.fhir.views import spark_runner\n",
+    "\n",
+    "# The Spark dataset containing FHIR data. This may be read-only to the user.\n",
+    "fhir_dataset = \"default\"\n",
+    "\n",
+    "# The Spark dataset where we will create views, value sets, and other derived tables\n",
```

Comment:
I think dataset for this case? i think colloquially, database referts to the entire Spark entity, whereas dataset refers to a collection of tables

Reply:
Hmm, this might be a terminology difference when using SQL. For example take a look at this `SHOW TABLE` [documentation](https://spark.apache.org/docs/3.3.0/sql-ref-syntax-aux-show-tables.html) where it talks about "database" and we say: `SHOW TABLES IN default;` in our case. Not a big deal, please feel free to resolve and leave this for future.

Reply:
Resolving this for now.